README.md

```diff
@@ -21,10 +21,48 @@ To install into `${HOME}/.wireshark/plugins`, run:
 For debugging with GDB or LLDB, use `cmake -DCMAKE_BUILD_TYPE=Debug ..` instead.
 
 On Linux and MacOS, you will need to have Wireshark installed with the epan
-headers. For building on Windows, I have no idea. Building a static library
+headers. Building a static library
 is currently possible but I'm terrible at CMake so, good luck.
 
 
+### On Windows
```

Comment:
Might be worth a Wiki page for building on Windows. I'm fine to move the Linux and Mac info over there too.